src/monta_mcp/__init__.py

```
@@ -1,2 +1,373 @@
+    try:
+        logger.debug(f"Calling async_stop_charge(charge_id={charge_id})")
+        result = await asyncio.wait_for(
+            client.async_stop_charge(charge_id),
+            timeout=10.0
+        )
+        logger.info(f"Successfully stopped charge {charge_id}")
+        # Convert dataclass to dict
+        return asdict(result)
+    except asyncio.TimeoutError:
+        error_msg = f"Request timed out after 10 seconds for charge_id={charge_id}"
+        logger.error(error_msg)
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)